changelog.json

```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.9.6-beta2": [
+      "[Fixed] Stashing dialog no longer hangs when intiating cherry-pick - #13419",
+      "[Fixed] Rebase no longer hangs after conflicts resolved when intiated through pull conflict error - #13204",
+      "[Fixed] Discarding files and repositories would fail to move items to trash - #13433",
+      "[Improved] Added check runs overall completeness indicator in check run popover header - #13423"
```

Comment:
```suggestion
      "[Improved] Added check runs overall progress indicator in check run popover header - #13423"
```

Reply:
Discussed in slack - not exactly a progress indicator since it is a pie chart of all statuses.